.github/workflows/containers.yaml

```diff
@@ -25,8 +25,8 @@ jobs:
       - name: Set Environment
         id: vars
         run: |
-          echo "::set-output name=tag::${GITHUB_REF#refs/*/}"
-          echo "::set-output name=revision::$(git rev-parse --short HEAD)"
+          echo "tag=${GITHUB_REF#refs/*/}" >> $GITHUB_OUTPUT
+          echo "revision=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
```

Comment:
Are we missing something? I'm still seeing some `set-output` warnings in the CI: https://github.com/trisacrypto/directory/actions/runs/3680285547

Reply:
It looks like those deprecation warnings are bubbling up from some of our dependencies; once they have updated their actions as we have done in this PR, they should hopefully all go away.